cfssl.tf

```diff
@@ -98,6 +76,27 @@ resource "aws_volume_attachment" "cfssl-data" {
   device_name = "/dev/xvdf"
   volume_id   = "${aws_ebs_volume.cfssl-data.*.id[count.index]}"
   instance_id = "${aws_instance.cfssl.*.id[count.index]}"
+
+  provisioner "remote-exec" {
+    when = "destroy"
+
+    inline = [
+      "sudo systemctl stop cfssl.service",
+      "sudo umount -d /dev/xvdf",
```

Comment:
changed to match the docs https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-detaching-volume.html